internal/provisioner/cluster_sli.go

```diff
@@ -12,6 +12,43 @@ import (
 	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
 )
 
+const objective = 99.5
```

Comment:
Merely an opinion. Do we wish to retrieve this value from the ENV? Because, if we modify this value in the future, a new version will be required.

Reply:
My guess is that if this value is modified, we will need to re-provision every cluster's CRDs and that will be made either: 1) a script by the SRE team 2) a reprovision of all clusters. And even if we use an environment variable, we'll need to restart/redeploy the provisioner cnc for it to take effect (unsure if we have something like reloader in place).

I don't see much value adding a new flag for this configuration at this point tbh, but I'd want to hear SRETeam opinion on the subject, since this request comes from them and will have more context on the matter.

Reply:
Since you made this a variable take the full path and make it also a flag imho.

Reply:
Two against one, huh? 😛

Reply:
Pease check: https://github.com/mattermost/mattermost-cloud/pull/777/commits/28bdec31ad7e21fac134c7064ceeebbbf8c06c5e